profiles/OshekharO.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/95137948/217157979-0b9ccbee-9415-471c-9849-d9796923c5c1.jpg",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/217171649-a9cfaca7-944b-46b9-b21a-8165bf98555c.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/217171649-a9cfaca7-944b-46b9-b21a-8165bf98555c.png",
```

Reply:
Sure